java/web/skeleton/Makefile

```diff
@@ -1,7 +1,7 @@
 projectDir := $(realpath $(dir $(firstword $(MAKEFILE_LIST))))
 os := $(shell uname)
-app_name = java-web
-tenant_name = $(TENANT_NAME)
+app_name = {{ name | default("java-web") }}
+tenant_name = {{ tenant | default("$(TENANT_NAME)") }}
```

Comment:
Should helm chart also be updated similar to go template?
Also, does it make sense to update project name in `settings.gradle` and similar files?

Reply:
I think I've updated helm charts, but will double check. Will have a look in settings.gradle

Reply:
ah, I've done go-web but not java-web